cypress/fixtures/objects/moduleReqId/moduleReqId.json

```diff
@@ -2676,6 +2676,68 @@
                     }
                 ]
             }
+        },
+        "UserInterest": {
+          "Content.requestUserInterest - Positive Scenario: with type interest and reason playlist": {
+            "req": [
```

Comment:
If there is no spec document, we should not have these test cases added in moduleReqId

Reply:
Sorry didn't get this comment.. README for the user interest is the spec doc right? am I missing something?